terraform/geomatch_app/alb/main.tf

```diff
@@ -86,7 +86,123 @@ resource "aws_alb" "this" {
   tags = {
     Project     = var.project
     Environment = var.environment
-    Name        = local.name_prefix
+    Name        = "${local.name_prefix}"
+  }
+}
+
+resource "aws_lb_target_group" "this" {
```

Comment:
The target Rstudio groups / listeners I think make more sense in the dev_environment module (which should already take the alb_module as an input). The ALB module shouldn't have to have knowledge of the downstream services. 

The cognito groups / listeners I think make sense in this module